lib/src/devices/cf201.dart

```diff
@@ -0,0 +1,88 @@
+library dslink.mamac.nodes.devices.cf201;
+
+import '../mamac_device.dart';
+import '../devices.dart';
+
+class CF201 extends MamacDevice {
+  static const String type = 'cf201';
+  static const String xmlFile = 'cf201xml.txt';
```

Comment:
This should always have a .xml extension. The same one you have in this case was for illistration only. Filename should be `cf201.xml`